.husky/forbidden-branches.sh

```diff
@@ -0,0 +1,10 @@
+# shellcheck disable=SC2006
+BRANCH=`git rev-parse --abbrev-ref HEAD`
```

Comment:
```suggestion
BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

I ran the shell script through the shellcheck program, it suggests we use this syntax. I guess there's a reason you use the backtick syntax?

Reply:
You are right! My IDE was warning me about it, but when I told it to fix it, it wasn't changed 🤔 

Let me fix this.

Reply:
Done! I've moved the script outside husky path since files there are supposed to match a git hook.